stable/artifactory/templates/artifactory-deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ .Values.art_name }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+spec:
+  replicas: {{ .Values.art_replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.art_name }}
+      annotations:
+        pod.beta.kubernetes.io/init-containers: '[{
+          "name": "remove-lost-found",
+          "image": {{ .Values.initContainerImage | quote }},
+          "command": ["rm", "-rf", "{{ .Values.art_persistence.mountPath }}/lost+found"],
```

Comment:
Why would you want to do this? Sounds like a hack to me. If `lost+found` gets in your way, why don't you use a subdirectory?